src/screens/UserPortal/Chat/Chat.tsx

```
@@ -74,6 +126,25 @@
     };
   }, []);
 
+  React.useEffect(() => {
+    if (filterType === 'all') {
+      chatsListRefetch();
+      if (chatsListData && chatsListData.chatsByUserId) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)